contracts/nns/config.yml

```diff
@@ -2,7 +2,7 @@ name: "NameService"
 supportedstandards: ["NEP-11", "NEP-22"]
 safemethods: ["balanceOf", "decimals", "symbol", "totalSupply", "tokensOf", "ownerOf",
               "tokens", "properties", "roots", "getPrice", "isAvailable", "getRecords",
-              "resolve", "version", "getAllRecords"]
+              "resolve", "version", "getAllRecords", "hasNeoRecord"]
```

Comment:
All reader methods should be added here, those returning iterators are safe as well.